src/dc_receive_imf.rs

```diff
@@ -531,7 +529,8 @@ async fn add_parts(
                 }
                 Err(err) => {
                     warn!(context, "Error in Secure-Join message handling: {}", err);
-                    return Ok(None);
+                    chat_id = Some(DC_CHAT_ID_TRASH);
+                    securejoin_seen = true;
```

Comment:
I was unsure here, but decided that it probably makes sense to mark the erroneous messaege as seen